lib/trailblazer/operation/public_call.rb

```diff
@@ -19,15 +19,22 @@ def call(options = {}, *args)
       call_with_public_interface(options, *args)
     end
 
-    def call_with_public_interface(*args)
-      ctx = options_for_public_call(*args, flow_options())
+    # Default {@activity} call interface which doesn't accept {circuit_options}
+    #
+    # @param [Array] args => [ctx, flow_options]
+    #
+    # @return [Operation::Railway::Result]
+    #
+    # @private
+    def call_with_public_interface(options, flow_options = {})
+      ctx = options_for_public_call(options, original_flow_options(flow_options))
 
       # call the activity.
       # This will result in invoking {::call_with_circuit_interface}.
       # last_signal, (options, flow_options) = Activity::TaskWrap.invoke(self, [ctx, {}], {})
       signal, (ctx, flow_options) = Activity::TaskWrap.invoke(
         @activity,
-        [ctx, flow_options()],
+        [ctx, original_flow_options(flow_options)],
```

Comment:
:thinking: Why are we calling `original_flow_options` twice (line 30 and 37)?

Reply:
Fixed it by calling it once.